.github/workflows/ci.yml

```diff
@@ -22,3 +26,28 @@ jobs:
         uses: coverallsapp/github-action@master
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
+  
+  analyze:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      security-events: write
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Run Slither
+        uses: crytic/slither-action@v0.2.0
+        id: slither
+        with:
+          node-version: 16
+          sarif: results.sarif
+          fail-on: none
```

Comment:
Does this make the build pass even if there are problems?

Reply:
Yes, the build will continue. We need it to let the SARIF upload step run if Slither finds issues.